### PR TITLE
MNT Use GitHub Actions CI

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/gha-ci/issues/11

CI run - https://github.com/creative-commoners/recipe-kitchen-sink/actions?query=branch%3Apulls%2F4.10%2Fmodule-standards

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## Silverstripe CMS kitchen sink recipe
 
-[![Build Status](https://app.travis-ci.com/silverstripe/recipe-kitchen-sink.svg?branch=4)](https://travis-ci.com/silverstripe/recipe-kitchen-sink)
+[![CI](https://github.com/silverstripe/silverstripe-versioned/actions/workflows/ci.yml/badge.svg)](https://github.com/silverstripe/silverstripe-versioned/actions/workflows/ci.yml)
```
Comment (ChloeSartorelli) [inline README.md]: Why versioned?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  # Every day at 3:00pm UTC
+  schedule:
+    - cron: '0 15 * * *'
+
+jobs:
+  ci:
+    name: CI
+    # Only run cron on the silverstripe account
+    if: (github.event_name == 'schedule' && startsWith(github.repository, 'silverstripe/')) || (github.event_name != 'schedule')
+    uses: silverstripe/gha-ci/.github/workflows/ci.yml@v1
```
Comment (ChloeSartorelli) [inline .github/workflows/ci.yml:16]: Do we need the slack integration defined in the travis config (do we know if that was even still doing anything)?

**Review by GuySartorelli — CHANGES_REQUESTED** [review]
We're missing at least these two jobs from the travis config:
- recipe-ccl unit tests
- recipe-solr-search unit tests

The following deprecated modules are being tested, and they shouldn't:
- cwp-recipe-cms
- cwp-recipe-search
- cwp-recipe-core

File: .github/workflows/ci.yml (hunk 2 above)
Comment (emteknetnz) [reply .github/workflows/ci.yml:16]: No

File: README.md (hunk 1 above)
Comment (emteknetnz) [reply README.md]: mistake, updated

**Comment by emteknetnz** [comment]
Have updated phpunit config file, correct jobs should get created now

**Review by GuySartorelli — APPROVED** [review]
Everything is super failing but I can't see anything wrong with the config so it can be treated with the other failures in https://github.com/silverstripe/gha-ci/issues/37

**Pull request merged**